README.md

```diff
@@ -34,7 +34,11 @@
    curl -sSL https://install.python-poetry.org | python3 -
    poetry install
    ```
-
+1. Install dependencies locally
+```bash
+pip install cryptography==36.0.2
+poetry install
+```
```

Comment:
I would move this part under Cleaning-up the environment to connect this "chapter" with Code Quality, but it is optional.